notebooks/official/generative_ai/text_embedding_api_semantic_search_with_scann.ipynb

```diff
@@ -0,0 +1,1153 @@
+{
```

Comment:
<p>use sentence case for headings</p><p> </p><p>Create embedding dataset</p><p> </p><p>The dataset is solely to demonstrate the use of the -&gt; The dataset demonstrates the use of the</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>